src/hooks/progress/index.ts

```diff
@@ -1,5 +1,15 @@
-export const TUTORIAL_PROGRESS_QUERY_ID = 'tutorialProgress'
+/**
+ * Constants
```

Comment:
Have added `const`s here so that queries can reliably be aware of each other, which feels important for batching, and for query invalidation.